Dalamud/Interface/ImGuiSeStringRenderer/Internal/SeStringRenderer.cs

```diff
@@ -91,13 +72,23 @@ private SeStringRenderer(DataManager dm, TargetSigScanner sigScanner)
             dm.Excel.GetSheet<UIColor>() ?? throw new InvalidOperationException("Failed to access UIColor sheet."));
         this.gfd = dm.GetFile<GfdFile>("common/font/gfdata.gfd")!;
 
-        // SetUnhandledExceptionFilter(who cares);
-        // _set_purecall_handler(() => *(int*)0 = 0xff14);
-        // _set_invalid_parameter_handler(() => *(int*)0 = 0xff14);
-        var f = sigScanner.ScanText(
-                    "ff 15 ?? ?? ?? ?? 48 8d 0d ?? ?? ?? ?? e8 ?? ?? ?? ?? 48 8d 0d ?? ?? ?? ?? e8 ?? ?? ?? ??") + 26;
-        fixed (void* p = &this.setInvalidParameterHandler)
-            *(nint*)p = *(int*)f + f + 4;
+        // TODO: fix this in lumina
+        var fixedTable =
+            (typeof(Lumina.Text.SeStringBuilder)
+             .Assembly
+             .GetTypes()
+             .FirstOrDefault(x => x.Namespace == "Lumina.Text.Parse" && x.Name == "MacroStringParser")
+             !.GetField("Digits", BindingFlags.Static | BindingFlags.NonPublic)
+             !.GetValue(null) as sbyte[])
+            .AsSpan();
+        fixedTable.Fill(-1);
+        fixedTable['_'] = fixedTable['\''] = -2;
+        for (var i = '0'; i <= '9'; i++)
+            fixedTable[i] = (sbyte)(i - '0');
+        for (var i = 'A'; i <= 'F'; i++)
+            fixedTable[i] = (sbyte)(10 + (i - 'A'));
+        for (var i = 'a'; i <= 'f'; i++)
+            fixedTable[i] = (sbyte)(10 + (i - 'a'));
```

Comment:
How quickly can this fix be upstreamed?

Reply:
This is included in Lumina [v4.2.1](https://github.com/NotAdam/Lumina/releases/tag/4.2.1), so this PR needs to be updated.